mkn.sh

```
@@ -0,0 +1,27 @@
+ARGS="${TEST} -P mkn.base=gpu_"
+[ -n "XFILE" ] && ARGS+=" -x ${XFILE}"
+
+set -x
+
+(
+  [ ! -f "bin/core/libphare_core.so" ] && (
+      mkn clean build -Oqp test_core -d google.test,+ -a "-fPIC"
+      mkn clean build -tOqp core -a "-fPIC" -x res/mkn/mpi
+  )
+
+  mkn clean build -p test_core ${ARGS} $@
```

Shellcheck

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)